src/cts/test/simple_test_hier_out.vok

```diff
@@ -0,0 +1,121 @@
+module test_16_sinks (clk);
+ input clk;
+
+
+ INV_X1 clkload0 (.A(clknet_1_1__leaf_clk));
+ CLKBUF_X3 clkbuf_1_1__f_clk (.A(clknet_0_clk),
+    .Z(clknet_1_1__leaf_clk));
```

Comment:
It would be interesting to see a case where the clock buffers are pushed down in the hierarchy.